test/unit/steps/representative/representative-details/RepresentativeDetails.test.js

```diff
@@ -67,22 +67,18 @@ describe('RepresentativeDetails.js', () => {
   });
 
   describe('get getCYAName()', () => {
+  const NAME = 'MR,HARRY-Kane,O`Brian';
     beforeEach(() => {
       representativeDetails.fields.name.first.value = '';
       representativeDetails.fields.name.last.value = '';
     });
 
-    [
-      'MR,HARRY,POTTER',
-      'mr,harry,potter',
-      'mR,haRRy,pOttEr',
-      'Mr,harry John,pOttEr'
-    ].forEach(item => {
-      it(`should normalise reps full name # ${item}`, () => {
-        representativeDetails.fields.name.title.value = item.split(',')[0];
-        representativeDetails.fields.name.first.value = item.split(',')[1];
-        representativeDetails.fields.name.last.value = item.split(',')[2];
-        expect(representativeDetails.CYAName).to.equal('Mr Harry Potter');
+     beforeEach(() => {
```

Comment:
you're not doing any setup, so no need for the beforeEach

Reply:
i tried it without the beforeEach and it seems to be causing issues on the pipeline